tests/openebs-pool-container-failure_test.go

```diff
@@ -80,10 +82,45 @@ var _ = BeforeSuite(func() {
 var _ = Describe("BDD of openebs pool container failure experiment", func() {
 
 	// BDD TEST CASE 1
+	rv1 := 0
+	rc1 := 0
 	Context("Check for the components of openebs", func() {
 
 		It("Should check for creation of runner pod", func() {
 
+			//Getting the Sum of Resource Version before Chaos
+			csp_rv, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get csp pods")
+			for _, v := range csp_rv.Items {
+				i1, _ := strconv.Atoi(v.ResourceVersion)
+				rv1 = rv1 + i1
+			}
+
+			fmt.Printf("Resource Version before chaos has been recorded\n")
+
+			//Getting the ContainerIDs of CSP pod Containers
+			k := 0
+			csp, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
```

Comment:
Why are you creating the same object again, You can reuse the CSP object which was created earlier ( line 92)